tests/unit/test_translator.py

```diff
@@ -0,0 +1,80 @@
+import pathlib
+from typing import Optional
+
+from yangify import translator
+
+from yangson.datamodel import DataModel
+
+
+BASE = pathlib.Path(__file__).parent
+dm = DataModel.from_file(
+    f"{BASE}/yang/simple/yang-library-data.json", [f"{BASE}/yang/simple/"]
+)
+
+
+test_data = {
```

Comment:
This looks like the same fixture from `test_parser.py` (lines 181-200). Can we define these fixtures centrally for use across various test modules?

Reply:
Don't like fixtures for data, makes things harder to read as now you have to navigate between files to understand the test. Also, tests are completely independent, they have the same data now because it was easier to write this way but if we find edge cases we want to test they may evolve independently.

Reply:
I accept your answer and will move on with my life.